test/mpfr.jl

```diff
@@ -857,6 +857,49 @@ end
     # Issue #33676
     @test trunc(UInt8, parse(BigFloat,"255.1")) == UInt8(255)
     @test_throws InexactError trunc(UInt8, parse(BigFloat,"256.1"))
+
+    # Test `InexactError` limits
+    for T in Base.BitInteger_types
+        # Use `@eval` and interpolate `$T` to make failed tests easy to read.
+        @eval begin
```

Comment:
Is the `@eval` necessary?

Reply:
Not strictly. I'll update to use a `@testset` for as this should provide the same context without requiring an `@eval`